code/game/objects/structures/catwalk.dm

```diff
@@ -83,4 +83,4 @@
 
 
 /obj/structure/catwalk/can_prevent_fall()
```

Comment:
eh.

Reply:
Because catwalks can't prevent falls from above. If it's returning true, one can stand on openspace if catwalk is underneath it.